kornia/utils/misc.py

```diff
@@ -51,3 +54,25 @@ def vec_like(n: int, tensor: torch.Tensor, shared_memory: bool = False):
 
     vec = torch.zeros(n, 1, device=tensor.device, dtype=tensor.dtype)
     return vec[None].expand(tensor.shape[0], n, 1) if shared_memory else vec[None].repeat(tensor.shape[0], 1, 1)
+
+
+def reduce_first_dims(x: Tensor, keep_last_dims: int = None) -> Tuple[Optional[Tensor], Optional[Size]]:
```

Comment:
```suggestion
def reduce_first_dims(x: Tensor, keep_last_dims: int) -> Tuple[Optional[Tensor], Optional[Size]]:
```
and in which case the output can be Optional?